modules/http/src/main/resources/META-INF/mule-module.properties

```diff
@@ -58,6 +58,28 @@ artifact.export.classPackages=org.mule.runtime.module.http.api,\
                               org.glassfish.grizzly.streams,\
                               org.glassfish.grizzly.threadpool,\
                               org.glassfish.grizzly.utils,\
-                              org.glassfish.grizzly.utils.conditions
+                              org.glassfish.grizzly.utils.conditions,\
+                              \
```

Comment:
This is used as a separator between packages exported from different artifacts so it's clear when you read the file.